lib/bill_forward/entities/payment_method.rb

```diff
@@ -1,27 +1,27 @@
 module BillForward
-	class PaymentMethod < MutableEntity
-  		@resource_path = BillForward::ResourcePath.new("payment-methods", "paymentMethod")
+  class PaymentMethod < MutableEntity
```

Comment:
Yeah, but auto formatting is SO useful (and makes merging and committing so easier since everyone uses the same exact style). Now I wonder if there is something similar to `.jshintrc` for ruby.